src/main/CMakeLists.txt

```diff
@@ -18,7 +18,16 @@ set_target_properties(collab-vm-server PROPERTIES
 	RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}
 )
 
+
 collabvm_targetize(collab-vm-server)
 
 # TODO: This binary should be the only target which gets installed,
-# 	sans plugin-headers.
\ No newline at end of file
+# 	sans plugin-headers.
+
+# Dynamic binaries are SO much fun!
+if (WIN32)
+add_custom_command(TARGET collab-vm-server POST_BUILD
```

Comment:
If you're REALLY gonna do this, 
use https://cmake.org/cmake/help/latest/module/GetPrerequisites.html

This is fragile, only works on MinGW, and depends on scripts I just nuked (because they deserved it.)
Wherehas GetPrerequisites works everywhere (although Windows is the only place we need to copy dlls because windows has to be special)

For vcpkg you'll have to maybe nuke this if that's detected (the Q&D way I will not merge because I'm sure there's some way that's infinitely better would probably involve regex testing CMAKE_TOOLCHAIN_FILE for `/scripts/buildsystems/vcpkg.cmake`) though because vcpkg will copy shared libraries automatically (on any platform)

Reply:
yeaaaaaaaaaah i had a feeling this would of been the thing i would of used would of been bombed for using add_custom_command instead of looking into cmake's (what now) more than a thousand functions?

Reply:
>use https://cmake.org/cmake/help/latest/module/GetPrerequisites.html

``
`CMake Error at src/main/CMakeLists.txt:29 (get_prerequisites):
  Unknown CMake command "get_prerequisites".
```

> Deprecated since version 3.16: Use file(GET_RUNTIME_DEPENDENCIES) instead.
Deprecated? I think theres an (undocumented) "removed"  missing from this.

also >file(GET_RUNTIME_DEPENDENCIES)
yeah i'm not following that deathtrap. (cannot tell difference between 32-bit and 64-bit DLLs, makes it suck. hard.)

Reply:
the alternative to most of this (except add_custom_command) is `$<TARGET_RUNTIME_DLLS:tgt>`
which has one sliiiiiiiight issue with it
> *New in version 3.21.*

also another problem with it:
is it meant to work? for me with or without list expansion it's empty

Reply:
Apparently I forgot to check if the "deathtrap" was revised, and as of writing, it was... in 3.21.
I should of probably mentioned this.